README.md

```diff
@@ -4,41 +4,58 @@
 
 This repository includes examples that showcase usage of TypeDB Clients in reading from and writing to a TypeDB knowledge graph.
 
+## Explore Schemas
+- [Catalogue of Life](biology/catalogue_of_life/schema.tql) — a database containing the information about the taxonomy of life
+on Earth, showcasing automated loading of data using [TypeDB-Loader.](https://github.com/typedb-osi/typedb-loader)
+- [GitHub](software/github/schemas/github-schema.tql) — load data from a live repository on GitHub or from a Vaticle GitHub snapshot, and get results via a custom GUI interface that
+uses the Java client to fetch the requested data.
+- [Phone Calls](telecom/phone_calls/schema.tql) — a database of customers of a fictional telecom company and calls they make.
+- [XCOM 2](gaming/xcom/schema.tql) — a database of interdependent research tasks in the game XCOM 2, featuring automatic inference of available
+research based on completed tasks and available items.
+
 ## Explore Client Examples
```

Comment:
From here down I'm trying to categorize the examples and highlight why someone would want to take a closer look at each one.